ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly-4.14-upgrade-from-stable-4.14.yaml

```diff
@@ -280,6 +538,17 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
+- as: gcp-ipi-proxy-oidc-arm-f28
```

Comment:
I think we can remove this one, because we have `gcp-ipi-proxy-private-arm-f60` covers `proxy` and we have `gcp-ipi-oidc-rt-fips-amd-f60` covers `cco-manual-workload-identity`